jiwonjang/app.js

```diff
@@ -36,8 +38,8 @@ app.get("/ping", (req, res) => {
   return res.status(200).json({ message: "pong" });
 });
 
-app.post("/users", async (req, res) => {
-  const { name, email, profileImage, password } = req.body;
+app.post("/users/signup", async (req, res) => {
+  const { name, email, profile_image, password } = req.body;
```

Comment:
```suggestion
  const { name, email, profileImage, password } = req.body;
```
javascript에서는 일반적으로 snake_case를 쓰지 않습니다. camelCase를 사용할 수 있도록 전반적으로 재수정 부탁드릴게요!